.pylintrc

```diff
@@ -134,7 +134,8 @@ disable=print-statement,
         too-many-arguments,
         missing-docstring,
         protected-access,
-        line-too-long
+        line-too-long,
+        too-many-locals
```

Comment:
Do you think this needs to be added?

Reply:
It was necessary for the complicated request handlers - it's been removed and a single line comment disabling the pylint warning has been put on the block in question